Nighttime-Operations/Observatory/DIMM/index.rst

```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope. 
+The rms of the fluctuations of the angular distance between the two images is proportional to the atmospheric optical turbulence, that in turn, allows to predict the seeing FWHM. 
+
+The main control software is divided into several programs. The main services are `ameba` that is in charge of the observation, synchronizing dome, weather service, 
+mount and camera control, and the `preat` program that converts the output of the `dimm_tool` into seeing values.
+
+.. figure:: ./_static/program_structure.png
+    :name: DIMM program structure
+
+    Structure of the DIMM software
+
+
+Below is a list of some of the most useful DIMM manuals:
+
+.. _DIMM General Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-general_spec-en_V1-1.pdf
+.. _DIMM Configuration: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-config_spec-en_V1-2.pdf
+.. _tt_master Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-tt-master_spec-en_V1-4.pdf
+.. _tt_meteo Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-tt-meteo_spec-en_V1-2.pdf
+.. _Preat Description: https://curl.sai.msu.ru/mass/download/doc/preat_desc2e.pdf 
+.. _DIMM Optical and mechanical design: https://curl.sai.msu.ru/mass/download/doc/description.pdf 
+.. _DIMM soft description: https://curl.sai.msu.ru/mass/download/doc/dimm_soft_description.pdf
+.. _DIMM Confluence Page: https://confluence.lsstcorp.org/pages/viewpage.action?spaceKey=LTS&title=DIMM+Tower+and+Equipment+-+Edison
+.. _DIMM Source Code: https://github.com/lsst-ts/ts_dimm_app 
+
+- Overview of the DIMM software: `DIMM General Spec`_
+
+- Configuration files for the DIMM programs, specially `ameba` and `dimm_tool`: `DIMM Configuration`_
+
+- High-level control of the DIMM software, `tt-master` program description: `tt_master Spec`_
+
+- A detailed description of the `preat` program and preat output: `Preat Description`_
+
+- Information on the different operating modes that might be useful for maintenance and troubleshooting: `DIMM soft description`_
+
+- Rubin's confluence page with information of the individual hardware components, history of Rubin's DIMM, SW installation instructions and other useful links: `DIMM Confluence Page`_
+
+- DIMM source code: `DIMM Source Code`_
+
+
+To start up the DIMM, please follow the procedure detailed in :ref:`DIMM Startup <DIMM-StartUp>`
```

Comment:
I wonder if this is better put after 1st sentence? 
I imagine most people on this page will be more interested on how to start it than the docs? 
Up to you....

Reply:
Yes, I think it's a great idea.